docs/kcMigration/sso-migration-notification.md

```diff
@@ -0,0 +1,69 @@
+
+### What is happening?
+
+Hi team, the KeyCloak SSO service is finally removing its *Pathfinder* sticker and getting a new name.  This change does not affect the service offering but does change how you will access it.  The name is changing from sso.pathfinder.gov.bc.ca to oidc.gov.bc.ca.  For more information on this Single Sign-On service, refer to [here](https://developer.gov.bc.ca/BC-Government-SSO-Service-Definition).
+
+### When?
+
+We are planning for an accelerated deployment cadence in order to remove blockers for the OpenShift 4 migration. This will require fast action from product teams, both from the initial functional validation activities, as well as notifying the platform-experience team immediately of any critical issues identified during testing.
+
+This service name migration will carry out in the sequence of dev, test and then prod. For each environment, here are the major steps:
+
+- Step 1 - Enable new service name `*.oidc.gov.bc.ca`.
+
+  - **Please note that we are not shutting down the original service name at `sso.pathfinder.gov.bc.ca`. Instead, both pathfinder and the new `oidc.gov.bc.ca` service name will be supported for four weeks/two sprints (or longer depends on the feedback), which allows product teams to update integrated applications and conduct thorough testing.**
+
+- Step 2 - Product teams verify existing `pathfinder` SSO services working fine
+- Step 3 - Product teams update application's SSO configuration to the new service and test out SSO integrations
+- Step 4 - Platform Services team work with product teams to troubleshot any issue found
+- Step 5 - Shutdown the pathfinder service name
+
+
+#### Dev SSO New Service Name Enable on `Sep 24th`
+As no production applications will be affected, new service name update will be scheduled during work hours. The expected SSO service disruption will last for 1.5 to 2 hours.
+
+1 week of time are set aside for Dev environment testing before the Test environment service name change.
+
+#### Test SSO New Service Name Enable on `Oct 1st`
+As no production application will be affected, new service name update will be scheduled during work hours. The expected SSO service disruption will last for 1.5 to 2 hours.
+
+1 week of time are set aside for Test environment testing before the Production environment service name change.
+
+#### Production SSO New Service Name Enable on `Oct 8th`
+Current SSO service disruption for the new production service name is scheduled for 5:30PM - 7:30PM (2 hours), on **October 8th, 2020**.
+
+
+### Will there be an impact on the Platform apps?
+
+Yes there will be service disruption when we enable the new SSO service name, but the effects are expected to be temporary and will last for about 10mins for each realm. This is caused by KeyCloak internal settings update from Platform Services team, if you are interested in more details, refer to [doc](./KeycloakMigration.md).
```

Comment:
is this in addition to the 1.5 - 2 hour service disruptions mentioned above?

Reply:
no it's included. the 1.5 to 2 hours includes the 10 mins disruption for each realm.